src/chain.js

```diff
@@ -137,23 +137,7 @@ process.on("message", function(message) {
   }
 });
 
-
-function sendHeartBeat() {
-  process.send({type: 'heartbeat'})
-}
-
-sendHeartBeat()
-
-// Keep an interval going so the process stays open.
-// We can use this interval to maintain a heartbeat
-setInterval(function() {
-  sendHeartBeat()
-}, 1000);
-
-process.on('uncaughtException', (err) => {
-  console.log(err.stack || err)
-  process.send({type: 'error', data: err.stack || err})
-});
+process.send({type: 'started'})
```

Comment:
It appears this event isn't actually used anywhere (correct me if I'm wrong). Is it needed?

Reply:
It is used to tell the parent process that we're initialized and it can carry on. See line 21 in the new version of this file.